app/livechat/imports/server/rest/rooms.js

```diff
@@ -49,3 +50,17 @@ API.v1.addRoute('livechat/rooms', { authRequired: true }, {
 		}
 	},
 });
+
+API.v1.addRoute('livechat/rooms/:_id', { authRequired: true }, {
```

Comment:
You don't need to create a new endpoint to get the `room data` since the `APIClient` perform the Oauth process, so you can call the regular Rocket.Chat endpoint to get the `room`.